.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,78 @@
+## Description
+
+Fixes #.
+
+Changes proposed in this pull request:
+
+## Type of change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Vulnerabilities update
+- [ ] Config changes
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires an admin update after deploy
+- [ ] Includes a database migration removing  or renaming a field
+
```

Comment:
Just a thought, it might be nice to have a section for "screenshots" if the changes are visual

Reply:
It's there at the end of the markdown file. https://github.com/freedomofpress/pressfreedomtracker.us/pull/1703/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R76

Reply:
Oh my bad, didn't see it so far down